cvat/apps/engine/models.py

```diff
@@ -54,6 +54,18 @@ def choices(cls):
     def __str__(self):
         return self.value
 
+class UploadedDataStorageLocationChoice(str, Enum):
```

Comment:
I will vote for a shorter name like StorageChoice or something like that.

Reply:
Renamed